orchestra/tests/helpers/fixtures.py

```diff
@@ -235,7 +240,8 @@ def setup_models(test_case):
     for task_slug, details in tasks.items():
         task = TaskFactory(project=test_case.projects[details['project_name']],
                            step_slug=test_case.test_step_slug,
-                           status=details['status'])
+                           status=details['status'],
+                           start_datetime=parse('2015-10-12T01:00:00+00:00'))
```

Comment:
...it will be pulled from settings as whatever is in settings, which is implicit rather than explicit :).  leaving it explicit.